server/manage.py

```diff
@@ -6,22 +6,5 @@
 cli = FlaskGroup(create_app=create_app)
 
 
-@cli.command()
-def recreate_db():
-    db.drop_all()
-    db.create_all()
-    db.session.commit()
-
-@cli.command()
-def create_db():
-    db.create_all()
-    db.session.commit()
-
-@cli.command()
-def seed_db():
-    db.session.add(Foo(string_field='Hello World'))
-    db.session.commit()
-
-
 if __name__ == '__main__':
     cli()
```

Comment:
We are not using `FlaskGroup` here. We should run the app using the standard `app.run()`. See [basic Flask app](https://www.codementor.io/@overiq/basics-of-flask-fzvh8ueed).